app/controllers/email_events_controller.rb

```diff
@@ -0,0 +1,45 @@
+class EmailEventsController < ApplicationController
+  before_action :event, except: :show
+
+  def index
+    @email_events = EmailEvent.all
+  end
+
+  def new
+    @email_event = EmailEvent.new
+  end
+
+  def create
+    @email_event = EmailEvent.new(create_params)
+    @email_event.event_id = @event.id
+
+    if @email_event.save
+      @email_event[:to].split(',').each do |e|
+        user = User.find_by(email: e)
+        if user.nil?
+          flash[:error] = 'Mail did not delivered to ' + e + ', because this email is not registered with TSP'
```

Comment:
I would rephrase the first part to "Email was not delivered to". And maybe put the second part into a separate sentence, like "This email is not registered with TSP."